### PR TITLE
Revised Wisconsin.md

### PR DESCRIPTION
**Pull request by ContributingThrowaway**
Made assorted revisions to Wisconsin.md, most notably splitting incident WI-Milwaukee-1 into two incidents and expanding it.

### CODE REVIEW

File: reports/Wisconsin.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ## Madison
 
-### Officer pepper-sprays protestor walking backwards with arms raised | May 30th
+### Officer pepper-sprays or maces protestor walking backwards with arms raised | May 30th
 
-Ten seconds into the video, in the bottom-left corner, an officer pepper-sprays a protestor who is walking backwards with his arms raised.
+Ten seconds into the video, in the bottom-left corner, an officer sprays a protestor who is walking backwards with his arms raised (presumably with mace or pepper spray).
 
-tags: pepper-spray, protestor
+tags: protestor, spray
```
Comment (emarcey) [inline reports/Wisconsin.md:7]: should include pepper-spray, mace

File: reports/Wisconsin.md [hunk 2]
```diff
@@ -31,18 +31,46 @@ id: wi-madison02
...
 ## Milwaukee
 
-### Police restrain man using knee on neck | June 2nd
+### Police on bikes restrain man using knee on neck | June 2nd
 
-Officers restrain a protester by placing a knee on his neck. Article explains incident occurred after two officers collided with each other on bicycles, after which they began striking protesters.
+Protesters were congregating at the intersection of North 6th Street and McKinley Avenue. A group of officers on bikes rode up to them. Two officers on bikes collided with one another; one officer fell to the ground. Officers then tackled a protestor (for reasons unknown) and attempted to take him into custody, with one officer placing a knee on his neck or head for one or two minutes. No officers intervened to get the knee off the man's neck; police formed a barrier to screen the arrest from other protestors and people filming. At the same time, police arrested one other man and hit a woman in the collarbone -- again, for reasons unknown. The man who had an officer's knee on his neck has sued.
 
-tags: knee-on-neck, arrest, tear-gas, bike, projectiles, protestor
+tags: knee-on-neck, arrest, bike, protestor
```
Comment (emarcey) [inline reports/Wisconsin.md:38]: Also just `knee`, `strike`, `tackle`

File: reports/Wisconsin.md [hunk 3]
```diff
@@ -31,18 +31,46 @@ id: wi-madison02
...
+
+Police claim they used force to disperse the crowd after rocks and glass were thrown at them. Police also said at the time that "Molotov cocktails" were thrown at them. They later walked that statement back to say that one Molotov Cocktail was thrown. They say it did not ignite. Media and protesters present say they saw water bottles being thrown, but not glass, rocks, or a Molotov cocktail. Video released by police clearly shows something being thrown at police, but doesn't show what it was. Police also produced a picture of a crushed water bottle and what looks like a water pump and implied it was the remnants of a Molotov cocktail.
+
+NOTE: this incident immediately follows WI-Milwaukee-2. Because the misconduct involved is substantially different, it has been given its own incident.
+
+tags: gas, protestor, rubber-bullets, tear-gas
```
Comment (emarcey) [inline reports/Wisconsin.md:59]: "gas" isn't a tag. Also add "shoot." Maybe we should add "smoke-canister" as well as a new tag

File: reports/Wisconsin.md [hunk 4]
```diff
@@ -31,18 +31,46 @@ id: wi-madison02
...
+
+Police claim they used force to disperse the crowd after rocks and glass were thrown at them. Police also said at the time that "Molotov cocktails" were thrown at them. They later walked that statement back to say that one Molotov Cocktail was thrown. They say it did not ignite. Media and protesters present say they saw water bottles being thrown, but not glass, rocks, or a Molotov cocktail. Video released by police clearly shows something being thrown at police, but doesn't show what it was. Police also produced a picture of a crushed water bottle and what looks like a water pump and implied it was the remnants of a Molotov cocktail.
+
+NOTE: this incident immediately follows WI-Milwaukee-2. Because the misconduct involved is substantially different, it has been given its own incident.
```
Comment (emarcey) [inline reports/Wisconsin.md:57]: nitpick: id should be all lowercase

File: reports/Wisconsin.md [hunk 5]
```diff
@@ -31,18 +31,46 @@ id: wi-madison02
...
+
+tags: gas, protestor, rubber-bullets, tear-gas
+
+id: wi-milwaukee-2
+
+**Links**
+
+* [Instagram post with assorted photos, videos, and description](https://www.instagram.com/p/CA_heaaHWrc/?igshid=m5v0m438blws)
```
Comment (emarcey) [inline reports/Wisconsin.md:65]: Holy shitfucking balls that's a lot of links. Nice work!

Comment (ContributingThrowaway) [reply reports/Wisconsin.md:65]: Thanks!

File: reports/Wisconsin.md (hunk 3 above)
Comment (ContributingThrowaway) [reply reports/Wisconsin.md]: I believe gas is listed on "possible_tags.md"

As for adding a smoke-canister tag: Probably! They seem to crop up regularly.

Comment (emarcey) [reply reports/Wisconsin.md]: You're right about the gas tag! I think that's good enough since smoke alone isn't really a big deal

**Pull request merged**